src/pages/LogIn/components/LoginForm.tsx

```diff
@@ -51,6 +54,7 @@ class LoginFormComponent extends React.Component<AllProps, IState> {
       );
     } catch (error) {
       this.setState({
+        errorMessage:(()=>error.message==='Bad server response!'?'Invalid Credentials':error.message)(),
```

Comment:
`error.message` is not a stable API. Use other fields of `error` instead, perhaps `error.response`
https://github.com/FNNDSC/fnndsc/blob/f9fda903aa791c484123e9b9ec186a7fd0c3ec3a/js/chrisAPI/src/request.js#L179